meetings/2024-12-18.md

```diff
@@ -0,0 +1,50 @@
+# Node.js  TypeScript team Meeting 2024-12-18
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=BX_ewVG1lyg>
+* **GitHub Issue**: <https://github.com/nodejs/typescript/issues/21>
+
+## Present
+
+* Typescript team: @nodejs/typescript
+* Marco Ippolito: @marco-ippolito
+* Rob Palmer: @robpalme
+* Jacob Smith @JakobJingleheimer
+* Augustin Mauroy @augustinmauroy
+
+## Agenda
+
+## Announcements
+
+Jakob: codemod for migration to strip types almost read, should be released soon
+
+* Extracted from **typescript-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/nodejs.org
+
+* Expand CODEOWNERS to each learn section owner/advocate [#7292](https://github.com/nodejs/nodejs.org/issues/7292)
+
+Augustin: Website team would like to add @nodejs/typescript as codeowner of the typescript content of the website for review
+Marco: no issue with that, we are already doing it to some degree
+Jacob is working on a guide this weekend. Publishing TypeScript: <https://github.com/nodejs/nodejs.org/pull/7279>
+
+### nodejs/typescript
+
+* Unflag `--experimental-strip-types` [#17](https://github.com/nodejs/typescript/issues/17)
+Marco: issue with –eval, unflagging –experimental-strip-types would cause a breaking change in case Node.js was fed with invalid syntax.
+Created a PR to keep the current behavior and added explicit input types for typescript syntax
+<https://github.com/nodejs/node/pull/56285>
+This should make unflagging possible.
+Before unflagging we should create some content on how to publish TypeScript packages, some part of it was done by the TypeScript team. It’s also important to document decision making about `node_modules`.
+Augustin: should we mention in the guide, to publish as `esm` or `cjs` or both?
+Marco: we should link back to Jakob article on js publishing article, it should be linked but more focused on the ts side rather than general.
+Marco: The item on the issue about frontend metaframeworks is currently irrelevant because we don’t allow type stripping in node_modules so they still have to go through a transpilation step.
```

Comment:
```suggestion

* Unflag `--experimental-strip-types` [#17](https://github.com/nodejs/typescript/issues/17) 
  * Marco: issue with –eval, unflagging –experimental-strip-types would cause a breaking change in case Node.js was fed with invalid syntax.
    * Created a PR to keep the current behavior and added explicit input types for typescript syntax
<https://github.com/nodejs/node/pull/56285>
    * This should make unflagging possible.
    * Before unflagging we should create some content on how to publish TypeScript packages, some part of it was done by the TypeScript team. It’s also important to document decision making about `node_modules`.
  * Augustin: should we mention in the guide, to publish as `esm` or `cjs` or both?
  * Marco: we should link back to Jakob article on js publishing article, it should be linked but more focused on the ts side rather than general.
  * Marco: The item on the issue about frontend metaframeworks is currently irrelevant because we don’t allow type stripping in node_modules so they still have to go through a transpilation step.
```